styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -314,6 +315,7 @@ static Pod createPod(WorkflowInstance workflowInstance,
         .addToAnnotations(STYX_WORKFLOW_INSTANCE_ANNOTATION, workflowInstance.toKey())
         .addToAnnotations(DOCKER_TERMINATION_LOGGING_ANNOTATION,
                           String.valueOf(runSpec.terminationLogging()))
+        .addToLabels(buildLabel(workflowInstance, runSpec, styxEnvironment))
```

Comment:
Do we want this to fail the pod creation?

Reply:
It should never ever fail the pod creation.

Reply:
Aa I thought it would fail pod creation as it was constructed.

Reply:
What would happen if buildLabel would throw an exception?

Reply:
If you are referring to the hash algorithm loading exception, that is just something unfortunate due to different editions of JDK. In standard JDK, it will be there.

Or anything else you refer to?

Reply:
ok, yes, and I was just thinking if there was any exception in any function. bytesToHex for example, but maybe its not common to guard for this.

Reply:
Guarding this to be super sure.